### PR TITLE
Point: Add missing icons to the GUI

### PR DESCRIPTION
**Pull request by luzpaz**
Point Icons (Convert, Merge, and Structure) designed by @bitacovir  
Forum thread: https://forum.freecadweb.org/viewtopic.php?f=34&t=52394

### CODE REVIEW

File: src/Mod/Points/Gui/Workbench.cpp [hunk 1]
```diff
@@ -57,6 +57,9 @@ Gui::ToolBarItem* Workbench::setupToolBars() const
     *pnt << "Points_Import"
          << "Points_Export"
          << "Separator"
+         << "Points_Convert"
+         << "Points_Structure"
+         << "Points_Merge"
```
Comment (luzpaz) [inline src/Mod/Points/Gui/Workbench.cpp:62]: Does this look correct?

File: src/Mod/Points/Gui/Workbench.cpp [hunk 2]
```diff
@@ -68,7 +71,10 @@ Gui::ToolBarItem* Workbench::setupCommandBars() const
     Gui::ToolBarItem* pnt = new Gui::ToolBarItem( root );
     pnt->setCommand("Points tools");
     *pnt << "Points_Import"
-         << "Points_Export";
+         << "Points_Export"
+         << "Points_Convert"
+         << "Points_Structure"
+         << "Points_Merge";
```
Comment (luzpaz) [inline src/Mod/Points/Gui/Workbench.cpp:77]: Does this look correct?

**Review by luzpaz — COMMENTED** [review]
Can a dev take a quick look and see if I did anything incorrectly in the code? Thanks!

File: src/Mod/Points/Gui/Workbench.cpp (hunk 1 above)
Comment (chennes) [reply src/Mod/Points/Gui/Workbench.cpp:62]: Looks right to me - it compiles fine, and the icons display.

**Review by chennes — APPROVED** [review]
This compiles and runs without any problems, and the new icons display correctly and look great.

**Pull request merged**